spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/KafkaBinderConfiguration.java

```diff
@@ -55,7 +56,7 @@
  */
 @Configuration
 @ConditionalOnMissingBean(Binder.class)
-@Import({ PropertyPlaceholderAutoConfiguration.class, KafkaBinderHealthIndicatorConfiguration.class })
+@Import({KafkaAutoConfiguration.class, PropertyPlaceholderAutoConfiguration.class, KafkaBinderHealthIndicatorConfiguration.class })
```

Comment:
Is this a consequence of the fact that we configure a Binder in the child context?
Thanks

Reply:
Yes, but most importantly this aligns it with the same way Rabbit binder is configured. See the actual issue for more details

Reply:
Right, and now I understand why we need it there and I know more about binders and their relationship with the parent application context.
Thanks
LGTM